### ISSUE TITLE
Failing kubectl skew test: Kubectl alpha client [k8s.io] Kubectl run ScheduledJob should create a ScheduledJob

### ISSUE THREAD

**Issue by pwittrock** [kubernetes/kubernetes#51048]
**TL;DR:** We should disable this test in the 1.7 branch since it is expected to fail against a 1.8 cluster

---

The following test is failing when running a 1.7 client against a 1.8 cluster on gce:

> Kubectl alpha client [k8s.io] Kubectl run ScheduledJob should create a ScheduledJob

Failure message:

> generator \"scheduledjob/v2alpha1\" not found

It looks like this maybe related to this [PR](https://github.com/kubernetes/kubernetes/pull/49930) to remove the ScheduledJob alpha API.

[Test grid](https://k8s-testgrid.appspot.com/sig-cli#skew-cluster1.7-kubectl1.8-gce) link to failure history

Test originally added here:
[commit](https://github.com/kubernetes/kubernetes/commit/f0b97b8ffa75939ed5515d27a2e77dfe4a91701a#diff-540d8a63e8b72b4e05dda54ae37f77d2)
[PR](https://github.com/kubernetes/kubernetes/pull/36021)

cc @kubernetes/sig-cli-test-failures 
cc @soltysh @fabianofranz

**Comment by zjj2wry** [comment]
@pwittrock this is kubectl special a not exist generator, the master e2e test has updated to 
cronjob/v2alpha1. need update 1.7 cli e2e from scheduledjob/v2alpha1 to cronjob/v2alpha1?

**Comment by pwittrock** [comment]
Is it updating the test or the cli itself?  I was thinking we just need to disable the test in the 1.7 branch.

**Comment by pwittrock** [comment]
Updated the description to reflect this.

**Comment by soltysh** [comment]
I've just spoke with Philip, @fabianofranz is going to disable that test in 1.7 branch. Thank you Fabiano ❤️ !!!

/assign @fabianofranz

**Comment by fabianofranz** [comment]
On it.  ;)

**Comment by jdumars** [comment]
Should this still be open, @pwittrock ?
